docs/posts/roadmap-2024-H1/index.qmd

```diff
@@ -0,0 +1,219 @@
+
+::: {.callout-note}
+This is a great opportunity for new contributors to get involved with Ibis! Let
+us know if you're interested in adding geospatial support to your favorite
+backend.
+:::
+
+### Community engagement
+
+Hello! Expect to see an increased presence from the Ibis project in the form of
+blogs, conference talks, video content, and more in 2024. [Join us on
+Zulip](https://ibis-project.zulipchat.com) to discuss ideas and get involved!
+
+We would love to onboard new contributors to the project.
+
+### New backends
+
+Adding new backends is not a priority for the Ibis team at Voltron Data in Q1.
+Instead, we are focusing on [the big refactor](#the-big-refactor) and other
+internal library improvements to get Ibis to the point where adding new backends
+is much easier and maintanable. That will take the form of stabilizing the new
+intermediary representation, separating out **connection** from **compilation**
+steps, and solidifying the API for backend authors. We will also introduce new
+documentation and possibly testing frameworks to ease the burden of adding new
+backends.
+
+We are still happy to support new backends! Some have already been mentioned,
+but being added in Q1 include:
+
+- Apache Flink
+- Exasol
+- RisingWave
+
+Adding a new backend is a great way to get involved with Ibis! If you're
+interested, [join us on Zulip](https://ibis-project.zulipchat.com) and let us
+know or [open an issue on
+GitHub](https://github.com/ibis-project/ibis/issues/new/choose).
+
+### Logo and website design
+
+We will likely engage an external design firm to help us redesign the logo
+(initially created by Tim Swast, thanks Tim! It has served us well!) and website
+theme. We aim to keep the website simple and focused on documentation that helps
+users, but want to deviate from the default themes in Quarto to make Ibis stand
+out.
+
+### Documentation
+
+> "When you're ~~selling~~ distributing free and open source software, the
+> documentation is the product." - old tech adage, origin unknown
+
+A few months ago, we moved our documentation to [Quarto](https://quarto.org) and
+revamped most of the website along the way. We will continue improving the
+documentation with backend-specific getting started tutorials, how-to guides for
+common tasks, improved API references, improving the website search
+functionality, and more!
+
+Improving the documentation is a great way to get involved with Ibis!
+
+## Beyond Q1 2024
+
+This writeup of our roadmap is heavily biased toward Q1 of 2024. Looking out,
+our priorities remain much the same. After the big refactor is done, we will
+continue improving our library internals, backend interface, and ensuring the
+longevity of Ibis. We'll continue improving ML, streaming, and geospatial
+support.
+
+Expect an updated roadmap blog in the second half of the year for more details!
+
+## Next steps
+
+It's never been a better time to get involved with Ibis. [Join us on Zulip and
```

Comment:
Let's spell out what "involved with Ibis" means. It can either be contributing new backends (for xxx value), or integrating Ibis into your own framework (for yyy value), or using ibis as a pandas replacement for (zzz value), etc.